MyBlog.Services/Areas/Users/UserService.cs

```diff
@@ -64,9 +64,10 @@ public async Task<UserDto> UpdateByIdAsync(int id, UserDtoInput userInput)
     {
         var user = await GetUserByIdAsync(id);
 
-        if (await CheckEmailAndPhoneForFreeAsync(id, userInput) == false)
+        var freeEmailAndPhone = await CheckEmailAndPhoneForFreeAsync(id, userInput);
+        if (freeEmailAndPhone == false)
         {
```

Comment:
Rule for bool:
start name it with "is", so bool variable will be answer isEmailAndPhoneFree ?